<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManagerModuleWidget</class>
 <widget class="QWidget" name="ManagerModuleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>810</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>815</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>791</width>
     <height>481</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>查询信息：     </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_item">
        <property name="styleSheet">
         <string notr="true">background-color:rgba(0, 255, 127, 200);
border-radius:5px;
font: 12pt &quot;楷体&quot;;
color:rgb(0,0,0);
padding:3px;</string>
        </property>
        <item>
         <property name="text">
          <string>学生</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>教师</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>管理员</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>课程</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="button_add">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgba(0, 255, 127, 200);
border-style:outset;
border-width:2px;
border-radius:5px;
border-color:rgba(255,255,255,30);
font: 12pt &quot;楷体&quot;;
color:rgb(0,0,0);
padding:3px;</string>
        </property>
        <property name="text">
         <string>添加</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_delete">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgba(0, 255, 127, 200);
border-style:outset;
border-width:2px;
border-radius:5px;
border-color:rgba(255,255,255,30);
font: 12pt &quot;楷体&quot;;
color:rgb(0,0,0);
padding:3px;</string>
        </property>
        <property name="text">
         <string>删除</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="button_save_change">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgba(0, 255, 127, 200);
border-style:outset;
border-width:2px;
border-radius:5px;
border-color:rgba(255,255,255,30);
font: 12pt &quot;楷体&quot;;
color:rgb(0,0,0);
padding:3px;</string>
        </property>
        <property name="text">
         <string>保存更改</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableView" name="tableView">
      <property name="styleSheet">
       <string notr="true">background-color: rgba(41, 139, 201, 150);
border-color: rgba(41, 139, 201, 150);
font: 12pt;
color: black;
</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
